modules/lambdas/variables.tf

```diff
@@ -112,14 +124,10 @@ variable "orca_recovery_retry_backoff" {
 
 ## OPTIONAL (DO NOT CHANGE!) - Development use only
 
-variable "database_name" {
-  type        = string
-  description = "Name of the ORCA database in PostgreSQL"
-}
-
 variable "orca_recovery_retrieval_type" {
```

Comment:
Possibly remove this variable and make it hardcoded in modules/lambda/main.tf on line 166

Reply:
hardcoded in line 184 in modules/lambda/main.tf

`      RESTORE_RETRIEVAL_TYPE   = "Standard"`